src/vulkan-renderer/world/cube.cpp

```diff
@@ -426,4 +427,43 @@ std::vector<PolygonCache> Cube::polygons(const bool update_invalid) const {
     collect(*this);
     return polygons;
 }
+
+std::shared_ptr<Cube> create_random_world(int max_depth, const glm::vec3 &position, unsigned int seed) {
+    static std::random_device rd;
+    static std::mt19937 mt(seed);
```

Comment:
Can you remove the static here?

Reply:
Sure, though we'll need to redesign the RNG. Maybe the best option would be to just seed with the random device (which I just noticed isn't used)?

Reply:
Yeah, put the rd into the mt instead of the seed, if the seed is not passed (or 0)?

Reply:
Done